packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -1000,6 +1000,9 @@ class NotifyingLogger extends Logger {
 
   @override
   bool get hasTerminal => false;
+
+  @override
+  void clear() => _messageController.add(LogMessage('status', globals.terminal.clearScreen() + '\n'));
```

Comment:
This does not have a terminal attached, see `hasTerminal` so you cannot use this here. Instead, no-op

Reply:
Okay thanks, I was curious when writing these two whether it would actually have any effect. Are you absolutely sure this does not affect the IDE output?

Reply:
it doesn't

Reply:
Okay, I'll change it now. Would you happen to know what I would need to do to clear the IDE output -- or if there already is a button to do this in supported IDEs?

Reply:
The daemon logger is design specifically to send structured data to IDEs for consumption. They might not expose this functionality because they already have their own logic for displaying log messages